### DISCUSSION TITLE
Welcome to OpenCiviWiki Discussions!

### DISCUSSION THREAD

**Discussion by brylie** [CiviWiki/OpenCiviWiki#994] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by shayan-cyber** [comment]
Hey everyone. I'm starting this discussion thread to introduce myself.

I'm Shayan Debroy , pursuing B.Tech in Electronics and Communication Engineering from National Institute Of Technology, Silchar(India). I taught myself Python and django and came towards web development only because of the quality resources made available by the open source community .
It is my turn to help the open source community. I want to start my open source journey with this wonderful project CiviWiki.
I'm inclined to learn new technologies and have good experience in web development with django.
Eagerly waiting to learn as well as help the open source community.

**Reply by brylie** [reply]
Welcome @shayan-cyber :smiley: 

I can help you get started with CiviWiki development or with development on another open-source project called "[Wagtail Social Network](https://github.com/brylie/wagtail-social-network)" until we [choose a better name](https://github.com/brylie/wagtail-social-network/discussions/24).

Are you able to set up a CiviWiki development environment, as [outlined in our contributing guide](https://github.com/CiviWiki/OpenCiviWiki/blob/develop/CONTRIBUTING.md#development)?

**Reply by shayan-cyber** [reply]
yes , I've set up the development environment

**Reply by brylie** [reply]
OK, great! Would you like to beta test the app to find existing bugs? That will help us benchmark our progress and plan the next development steps.

**Reply by shayan-cyber** [reply]
yep, I'll be more than happy. Can you explain some of the major features of this project. Actually I checked the documentation but could not understand it properly

**Reply by brylie** [reply]
Sure. The main idea behind CiviWiki is to *foster constructive discussions around societal issues*. In that sense, the main features are "topics" and "threads," where threads are discussions around the topics.

None of the features are set in stone, and the names of things are changing. If features don't foster constructive discussions, they are probably not well-suited for the project.

**Reply by AbhijithGanesh** [reply]
@Brylie, I've written a blog on my page about my Open Source journey and I wanted to express my gratitude for your efforts and mentorship.
Feel free to check it out (I've mentioned you and I hope you won't mind it)
https://blog.abhijithganesh.tech

**Comment by gorkemarslan** [comment]
🎃 💻 🎃 💻 🎃 💻 🎃 💻 🎃 💻 🎃 💻 

Don't forget to register for [hactoberfast](https://hacktoberfest.digitalocean.com/)!

🎃 💻 🎃 💻 🎃 💻 🎃 💻 🎃 💻 🎃 💻

**Reply by brylie** [reply]
Let's discuss Hacktoberfest in a dedicated thread: #1076

**Comment by AbhijithGanesh** [comment]
@brylie is this project registered for Hacktoberfest?

**Reply by brylie** [reply]
Yes, we have added the `hacktoberfest` topic to the repo and are using the `hacktoberfest-accepted` label where appropriate. There isn't much else for projects to do in order to register for Hacktoberfest.

However, participants, like yourself, need to do a bit more to [register for Hacktoberfest](https://hacktoberfest.digitalocean.com/register).

**Reply by AbhijithGanesh** [reply]
No but in my dashboard it says project is not participating

**Reply by AbhijithGanesh** [reply]
![image](https://user-images.githubusercontent.com/67182544/135634192-72a262dc-8857-4e38-987d-ceefb2adcba7.png)

**Reply by brylie** [reply]
Let's discuss Hacktoberfest in a dedicated thread: https://github.com/CiviWiki/OpenCiviWiki/discussions/1076

**Comment by gorkemarslan** [comment]
Hactoberfest [FAQ](https://hacktoberfest.digitalocean.com/faq):

**Question:**
> What should I do if I contribute to a repository that is not participating in Hacktoberfest?

**Answer:**
>You can politely request the maintainer of the project you contributed to, to tag your contribution with the label **hacktoberfest-accepted** . Or, add the 'hacktoberfest' topic to the repository to opt the project into Hacktoberfest.
Any pull request with the **hacktoberfest-accepted** label, submitted to any public GitHub/GitLab repository, with or without the hacktoberfest topic, will be considered valid for Hacktoberfest.


**Question:**
> What are the rules?

**Answer:**
> To win a reward, you must sign up on the Hacktoberfest site and make four pull requests on any repositories classified with the 'hacktoberfest 'topic on GitHub or GitLab by October 31.  
> 
> Additionally, any pull request with the **hacktoberfest-accepted** label, submitted to any public GitHub repository, with or without the 'hacktoberfest' topic, will be considered valid for Hacktoberfest.


Our repository and the PR you created have already satisfied these rules. There might be some delay to process the requests from all participants. (I hope it does) @AbhijithGanesh, could you check it again?

**Reply by AbhijithGanesh** [reply]
I shall check and get back Gorkem Arslan

**Reply by gorkemarslan** [reply]
I guess your PR #1048 did not count unfortunately because you created the PR before Oct 1. I'm in the same situation. My pending PR, which is a draft, was created before Oct 1. 

I saw that in FAQ:

**Question:**

> I signed up for Hacktoberfest mid-October. Will pull requests that I created earlier in October count?

**Answer**

> Yes, all pull requests created between Oct 1 and Oct 31 will count, regardless of when you register for Hacktoberfest. 
> Pull requests created before Oct 1 but merged or marked as ready for review after do not count.

**Reply by brylie** [reply]
Let's discuss Hacktoberfest in a dedicated thread: https://github.com/CiviWiki/OpenCiviWiki/discussions/1076

**Comment by AbhijithGanesh** [comment]
Heyy @brylie can my name be updated for the other contributions I have made please? Thank you

**Reply by brylie** [reply]
@AbhijithGanesh you're listed as having contributed code, project management, and documentation.

![Peek 2021-10-02 10-45](https://user-images.githubusercontent.com/17307/135708211-13086e1f-4c87-4562-a0cd-3306529a3e43.gif)

What other types of contributions would you like added?

**Reply by AbhijithGanesh** [reply]
I would like to get the role which is related to bugs @brylie

**Reply by brylie** [reply]
Ok. Which bug report did you submit? I can add your bug badge in the bug report.

**Reply by AbhijithGanesh** [reply]
https://github.com/CiviWiki/OpenCiviWiki/issues/1022

**Reply by AbhijithGanesh** [reply]
@brylie you can check that out!

**Reply by brylie** [reply]
Done.

**Comment by AbhijithGanesh** [comment]
@brylie can we have PRs merged today?

**Reply by brylie** [reply]
@AbhijithGanesh, we need to keep discussions and issues focused on the relevant topic. The topic here is welcoming people to the CiviWiki discussions.

Please open a new discussion or issue to discuss other topics.

**Comment by Vinit-Kumar-Shah** [comment]
While Setting up the deployment server for the project I encountered the following project when I ran it on my local server. I would really appreciate it if someone can help me with this.
![image](https://user-images.githubusercontent.com/64884766/137027847-8f3d638c-ab08-41df-9599-a15a7d73fcf8.png)
![image](https://user-images.githubusercontent.com/64884766/137028459-9f4abd7a-b4a4-4701-b8c8-66a9279668ff.png)
P.S.: I think there is something wrong with the how_it_works view function but I haven't changed any code till now

**Reply by gorkemarslan** [reply]
@Vinit-Kumar-Shah, thank you for your interest. The issue will be solved after merging PR #1132. It would be better if you create a new issue for the next time.

In `discussions` section, we can talk about everything but [issues](https://github.com/CiviWiki/OpenCiviWiki/issues) section is just for this purpose. You can learn how to create a new issue with this [tutorial](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue). (In the case you don't know)

**Reply by Vinit-Kumar-Shah** [reply]
Thanks for replying @gorkemarslan I appreciate it.
Well, I do know about the issues section but the thing is I thought that there may be something that I might be doing wrong in the setup part so I haven't raised an issue. I'll keep that in mind from next time.

**Reply by brylie** [reply]
PR #1132 was merged. I'll mark this discussion as resolved.
